lib/util/template/index.ts

```diff
@@ -125,7 +125,7 @@ export const allowedFields = {
   prettyNewMajor: 'The new major value with v prepended to it.',
   prettyNewVersion: 'The new version value with v prepended to it.',
   project: 'ChangeLogProject object',
-  recreateClosed: 'If true, this PR will be recreated if closed',
+  recreateWhen: 'If "always", this PR will be recreated if closed',
```

Comment:
```suggestion
  recreateClosed: 'This PR will be recreated if closed',
```

Reply:
This value should then be massaged so that it's available to templates

Reply:
Could you give an example of what you mean? I couldn't find an example of `recreateClosed` being used in a template so couldn't think more on it.

Also do you mean `recreateClosed` should still be available for templates? But we renamed it so shouldn't it be replaced with `recreateWhen`.

Reply:
`recreateClosed` should still be available to templates.

It's not important to the user whether the original value was always/auto/never, but whether in the end the PR will be recreated if closed or not